hadoop-hdds/docs/content/tools/_index.zh.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "工具"
+date: "2017-10-10"
+summary: Ozone 为开发者提供一系列方便的工具，本页给出了命令行工具的简要列表。
+menu:
+   main:
+      weight: 8
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 有一系列管理 Ozone 的命令行工具。
+
+所有的命令都通过 ```ozone``` 脚本调用。
+
+守护进程命令：
+
+   * **scm** - 启动或停止 Storage Container Manager。
+   * **om** -  启动或停止 Ozone Manager。
+   * **datanode** - 启动或停止数据节点。
+   stopped.
+   * **s3g** -
+
+客户端命令：
+
+   * **sh** -  操作 Ozone 中的卷、桶和键的主要命令。
+   * **fs** - 运行一个文件系统命令（类似于 `hdfs dfs`）
+   * **version** - 打印 Ozone 和 HDDS 的版本。
+
+
+管理命令：
+
+   * **admin** -  Collects admin and developer related commands related to the 
```

Comment:
I am not sure how to translate this since I cannot use admin command in apace/ozone docker image

Reply:
Admin commands are refactor since the last docker image. They required dev build IMHO.

I uploaded a recent build to docker hub, you can try this command with:

```
docker run -it elek/ozone-dev:4b1fa1099 bash
ozone admin --help
```

Does it help?

(Let me know if I misunderstood the problem)

Reply:
Yes, this is what I am asking, but I cannot pull this image and I find that last image you uploaded is two months ago (https://hub.docker.com/r/elek/ozonedev/tags)

Reply:
Ups, sorry. I tried to do it in a new way with a script and didn't notice that the publish step is failed. 

Should be good now (note: `elek/ozone-dev`, and not `elek/ozonedev`. It's just a sandbox, and my naming is not consistent....) 

https://hub.docker.com/layers/elek/ozone-dev/4b1fa1099/images/sha256-ca96c5241cd291d8da11349632b9653cfd51ad4dc52c602062af21d8f881be0f?context=explore

Reply:
Thanks @elek . I have tried admin command and updated the translation doc.